python/ray/dashboard/modules/train/train_head.py

```diff
@@ -244,9 +241,7 @@ async def _get_run_status(
         # train controller.
         # We need to detect this case and mark the train run as ABORTED.
 
-        actor_infos = await DataOrganizer.get_actor_infos(
-            actor_ids=[train_run.controller_actor_id],
-        )
+        actor_infos = await self._get_actor_infos([train_run.controller_actor_id])
```

Comment:
is the format the same between what returns from http and what returns from DataOrganizer directly?

Reply:
Yes. They are the same.